tests/monitor/manager/test_builder.py

```diff
@@ -0,0 +1,90 @@
+import os
+import pytz
+from datetime import datetime
+
+import pytest
+
+from whylabs_toolkit.monitor.models import *
+from tests.helpers.test_monitor_helpers import BaseTestMonitor
+
+
+def test_set_fixed_dates_baseline(builder):
+    builder.set_fixed_dates_baseline(
+        start_date=datetime(2023,1,1),
+        end_date=datetime(2023,1,2)
+    )
+
+    assert builder.config.baseline == TimeRangeBaseline(
+        range=TimeRange(
+            start=datetime(2023,1,1, tzinfo=pytz.utc),
+            end=datetime(2023,1,2, tzinfo=pytz.utc)
+        )
+    )
+
+def test_exclude_target_columns(builder):
+    builder.exclude_target_columns(
+        columns=["prediction_temperature"]
+    )
+
+    assert builder._exclude_columns == ["prediction_temperature"]
+
+
+def test_set_target_columns(builder):
+    builder.set_target_columns(
+        columns=["prediction_temperature"]
+    )
+
+    assert builder._target_columns == ["prediction_temperature"]
+
+
+def test_build(builder):
+    assert not builder.monitor
+    assert not builder.analyzer
+
+    builder.build()
+
+    assert isinstance(builder.monitor, Monitor)
+    assert isinstance(builder.analyzer, Analyzer)
```

Comment:
This part seems a little suspect to me. Does this mean that `builder.build()` has a side effect that changes `builder.monitor`? It would be best if it was just a function that returned something new if thats the case.

Reply:
calling it `MonitorSetup` now! lmk if this works